reviews/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from reviews.models        import Review
+from django.http           import JsonResponse, request
+from django.views          import View
+from users.models          import User
+from products.models       import Product
+from users.utils           import login_decorator
+
+class ReviewView(View):
+    @login_decorator    
+    def post(self,request):
+        try:
+            data = json.loads(request.body)
+
+            if data["title"] == "" :
+                return JsonResponse({"MESSAGE":"NULL_REVIEWS"}, status = 400)
+
+            if data["size_rating"] == "" or data["color_rating"] == "" or data["delivery_rating"] == "" :
+                return JsonResponse({"MESSAGE":"NULL_RATING"}, status = 400)
+
+            rating_max = 6
+            if data["size_rating"] >= rating_max or data["color_rating"] >= rating_max or data["delivery_rating"] >= rating_max :
```

Comment:
```suggestion
            RATING_MAX = 6
            
            if data["size_rating"] >= rating_max or data["color_rating"] >= rating_max or data["delivery_rating"] >= rating_max :
```
상수화 할 때는 대문자로만 문자를 만듭니다.
그리고 상수랑 if 사이에 공백 하나 추가해 주세요.